core/templates/dev/head/pages/exploration_editor/statistics_tab/StatisticsTab.js

```diff
@@ -59,7 +61,12 @@ oppia.controller('StatisticsTab', [
           '/createhandler/data/' + explorationData.explorationId);
 
         $http.get(explorationDataUrl).then(function(response) {
-          var states = response.data.states;
+          var statesDict = response.data.states;
+          var states = {};
+          for (var stateName in statesDict) {
```

Comment:
Hm, and here it is again. I am starting to lean more strongly towards having an object for it...